{
	"name": "CVE-2024-0654",
	"description": "A vulnerability, which was classified as problematic, was found in DeepFaceLab pretrained DF.wf.288res.384.92.72.22. Affected is an unknown function of the file mainscripts/Util.py. The manipulation leads to deserialization. Local access is required to approach this attack. The exploit has been disclosed to the public and may be used. VDB-251382 is the identifier assigned to this vulnerability.",
	"type": "local",
	"severity": "none",
	"state": "published",
	"references": [
		"https://github.com/bayuncao/vul-cve-4",
		"https://github.com/bayuncao/vul-cve-4/blob/main/picture/1071705290840_.pic_hd.jpg",
		"https://vuldb.com/?ctiid.251382",
		"https://vuldb.com/?id.251382"
	],
	"datetime": "2025-02-06 12:34:59",
	"connections": [],
	"devices": [],
	"distributions": [],
	"packages": [],
	"products": [
		{
			"name": "iperov:deepfacelab",
			"vendor": "iperov",
			"product": "deepfacelab",
			"version": "df.wf.288res.384.92.72.22",
			"type": "software"
		}
	],
	"programs": [],
	"weaknesses": [
		{
			"name": "CWE-502",
			"impact": "any",
			"scope": "any"
		}
	]
}